Data/Sys/GameSettings/SMN.ini

```diff
@@ -5,7 +5,7 @@
 
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
-EmulationIssues = Enable EFB to RAM for the coins to spin (it will cause a big slowdown).
+EmulationIssues = Efb to texture needs scaled efb copies to be disabled for the coins to spin.
```

Comment:
We don't use the EFB to texture terminology anymore. My suggestion for this line is "If Store EFB Copies to Texture Only is enabled, Scaled EFB Copy needs to be disabled for the coins to spin."